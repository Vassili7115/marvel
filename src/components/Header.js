import React from 'react';
import '../stylesheets/header.scss'

const Header = () => {
        return(
            <div className="header-container">
                <div className="logo-container">
                    <a href="/"><img className="logo-header" src="/images/logo-marvel-studios.png" alt="marvel-logo"/></a>
                </div>
                <div className="menu-container">
                    <a href="/">Home</a>
                    <a href="/characters">Characters</a>
                    <a href="/comics">Comics</a>
                    <a href="/series">Series</a>
                </div>
            </div>
        )
    }   

export default Header;
