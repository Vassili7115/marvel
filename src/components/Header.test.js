import React from 'react';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';

import Header from './Header';

Enzyme.configure({ adapter: new Adapter() });

describe('Header component', () => {
  test('renders', () => {
    const wrapper = shallow(<Header />);

    expect(wrapper.exists()).toBe(true);
  });
});

it('renders correctly', () => {
  const tree = renderer
    .create(<Header />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
