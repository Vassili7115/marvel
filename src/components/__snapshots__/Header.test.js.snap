// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="header-container"
>
  <div
    className="logo-container"
  >
    <a
      href="/"
    >
      <img
        alt="marvel-logo"
        className="logo-header"
        src="/images/logo-marvel-studios.png"
      />
    </a>
  </div>
  <div
    className="menu-container"
  >
    <a
      href="/"
    >
      Home
    </a>
    <a
      href="/characters"
    >
      Characters
    </a>
    <a
      href="/comics"
    >
      Comics
    </a>
    <a
      href="/series"
    >
      Series
    </a>
  </div>
</div>
`;
